plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/sessionconfig/CodeScanSessionConfig.kt

```diff
@@ -174,7 +175,7 @@ class CodeScanSessionConfig(
             val current = stack.pop()
 
             if (!current.isDirectory) {
-                if (!featureDevSessionContext.ignoreFile(current)) {
+                if (runBlocking { !featureDevSessionContext.ignoreFile(current) }) {
```

Comment:
if codescan is also collecting files into a zip file, not sure why it's not just the same logic to make a zip file.

again needs followup

Reply:
sure